server/src/streaming/systems/stats.rs

```diff
@@ -32,7 +36,7 @@ impl System {
                 .map(|s| {
                     s.topics
                         .values()
-                        .map(|t| t.partitions.len() as u32)
+                        .map(|t: &crate::streaming::topics::topic::Topic| t.partitions.len() as u32)
```

Comment:
Make it shorter and move `crate::streaming::topics::topic::Topic` to `use` block